schema_based_on_diagram.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE patients(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(250),
+  PRIMARY KEY (id)
+);
```

Comment:
- [ ] Great job on the project requirements but you missed adding `date_of_birth` in the `patients` table as on the diagram. Please make sure to add that.

![image](https://github.com/malikhaiderkhan/Vet-Clinic/assets/83803414/35f0083b-5a8f-44a2-8fc6-7f250ea33e9d)